schema/EARMARK_MEMO.json

```diff
@@ -498,7 +498,7 @@
             ],
             "minLength": 0,
             "maxLength": 9,
-            "pattern": "^C[0-9]{0,9}$",
+            "pattern": "^C[0-9]{8}$",
```

Comment:
should we update the spec? https://docs.google.com/spreadsheets/d/1SRS_hfYEHQl13aoC-z1xpxubfbKKNMSr/edit#gid=329261904
In other transaction types the pattern is `^(?:[PC][0-9]{8}|[HS][0-9]{1}[A-Z]{2}[0-9]{5})$`
https://docs.google.com/spreadsheets/d/1SRS_hfYEHQl13aoC-z1xpxubfbKKNMSr/edit#gid=559306225 those might need to be updated

Reply:
We need to run all this by Aurelia to make sure it is the correct thing to do. I will discuss it with her in the morning.